### ISSUE TITLE
Platform view accessibility does not respond to taps on Moto G 4

### ISSUE THREAD

**Issue by matthew-carroll** [flutter/flutter#35432]
Running the webview example project on a Moto G 4, tapping on elements within the webview does not focus them. Instead, to give talk back focus to anything within the webview, I have to swipe left/right. Tapping on Flutter widgets, however, does correctly focus them for talk back.

The Moto G 4 that I'm using is running Android 7.0.

**Comment by matthew-carroll** [comment]
CC @amirh

**Comment by amirh** [comment]
@mklim

**Comment by mklim** [comment]
At a first glance it's worth pointing out that this is Android **N**, so probably unrelated to the brittle workarounds we've needed to add post O. Probably some other problem with the a11y ID behavior though.

**Comment by TahaTesser** [comment]
Hi @matthew-carroll  @mklim 
Is this issue still valid?
Thank you

**Comment by darshankawar** [comment]
> The Moto G 4 that I'm using is running Android 7.0.

@matthew-carroll Hello, this particular model seems to be discontinued now. Is there a way for you to confirm if the original issue / behavior you reported still occurs with latest stable ?

**Comment by matthew-carroll** [comment]
No. This issue is 5 years old. I'm sure at the time I was using a device from the Flutter team's test devices. Please follow up with current team members to see what makes sense in terms of repro.
